src/Containers/Impl/Docker/LocalDockerContainer.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.R.Containers.Docker {
+    public class LocalDockerContainer : IContainer {
+        public string Id { get; set; }
```

Comment:
Id prob never change during lifetime, I'd prefer ctor with parameter and just `get`